robottelo.properties.sample

```diff
@@ -26,6 +26,8 @@ project=sat
 locale=en_US.UTF-8
 remote=0
 smoke=0
+# Update upstream=0 for downstream run
+upstream=1
```

Comment:
In my experience, boolean flags are often quickly replaced with more generic settings. What if we end up with a third test mode beyond just upstream/downstream? For example, might we add a "rhai" test mode in the future? Doing something like this makes the setting both more flexible in the face of changing requirements:

    # Legal values are 'upstream' and 'downstream'.
    product=upstream

Thoughts?